### DISCUSSION TITLE
支持 Azure OpenAI Service

### DISCUSSION THREAD

**Discussion by Loongphy** [immersive-translate/immersive-translate#423] [Ideas]
### 新功能 | New Feature

支持Azure OpenAI或者支持自动义OpenAI URL

- Azure REST API
https://learn.microsoft.com/zh-cn/azure/cognitive-services/openai/reference#completions

### 为什么您想要添加这些功能 | Why you want to add these features

OpenAI 付费太困难，Azure 更简单

**Comment by theowenyoung** [comment]
自定义azure api 已支持了哈，直接填写 azure的key和url就可以了。

**Comment by Loongphy** [comment]
抱歉，文档没看全
